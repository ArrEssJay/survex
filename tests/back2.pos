( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) 1
(    0.00,    10.00,     0.00 ) 2
