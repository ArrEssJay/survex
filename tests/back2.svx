; pos=yes warn=6
*fix 1 0 0 0
*data normal from to tape compass backcompass clino backclino
1 2a 10.00 001.5 178.5 01 01
1 2b 10.00 358.5 181.5 01 01

*calibrate backcompass 10
*data normal from to tape compass backcompass clino backclino
1 3a 10.00 001.5 188.5 01 01
1 3b 10.00 358.5 191.5 01 01

*calibrate compass 10
*calibrate backcompass 0
*data normal from to tape compass backcompass clino backclino
1 4a 10.00 011.5 178.5 01 01
1 4b 10.00 008.5 181.5 01 01

*data cartesian from to dx dy dz
; Work around variation in which station is picked for the maximum:
1 max 9 12 36
