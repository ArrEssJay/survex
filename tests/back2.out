./back2.svx:4: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg
./back2.svx:5: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg
./back2.svx:9: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg
./back2.svx:10: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg
./back2.svx:15: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg
./back2.svx:16: warning: COMPASS reading and BACKCOMPASS reading disagree by 3dg

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 7 survey stations, joined by 6 legs.
There is 1 loop.
Survey has 2 connected components.
Total length of survey legs =   60.00m (  60.00m adjusted)
Total plan length of survey legs =   60.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from 4b at 0.00m to 4b at 0.00m)
North-South range = 10.00m (from 4b at 10.00m to 1 at 0.00m)
East-West range = 0.00m (from 4b at 0.00m to 3b at -0.00m)
   6 1-nodes.
   1 6-node.
There were 6 warning(s).
